### DISCUSSION TITLE
Elección de dataset

### DISCUSSION THREAD

**Discussion by Anishoond** [rramosp/ai4eng.v1#236] [UDEA Students]
Cordial saludo profesor y monitores.

Es para saber si este dataset que queremos escoger para el proyecto nos puede servir, el dataset de test cumple con todos los requisitos pero el dataset de prueba con el que vamos a entrenar el código solo tiene 137 filas por lo que nos genero la duda de si podemos usar esta competencia para realizar el proyecto. 

Agradezco de antemano su atención.

Este es nuestro dataset: https://www.kaggle.com/competitions/restaurant-revenue-prediction

**Comment by rramosp** [comment]
hola anishood ... 137 filas se quedan demasiado cortas ... escoged otro por
favor.

On Thu, Mar 2, 2023 at 9:09 AM Anishoond ***@***.***> wrote:

> Cordial saludo profesor y monitores.
>
> Es para saber si este dataset que queremos escoger para el proyecto nos
> puede servir, el dataset de test cumple con todos los requisitos pero el
> dataset de prueba con el que vamos a entrenar el código solo tiene 137
> filas por lo que nos genero la duda de si podemos usar esta competencia
> para realizar el proyecto.
>
> Agradezco de antemano su atención.
>
> Este es nuestro dataset:
> https://www.kaggle.com/competitions/restaurant-revenue-prediction
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/236>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIID7R6LPVMVQYDYA6DW2CSYDANCNFSM6AAAAAAVNNVXLA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
